packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -28,6 +28,10 @@ $mdc-textfield-divider-on-light: rgba(black, .12);
 $mdc-textfield-divider-on-dark: rgba(white, .12);
 $mdc-textfield-disabled-border-on-light: rgba(#231f20, .26);
 $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
+$mdc-textfield-underline-on-light-idle: rgba(black, .5);
```

Comment:
You list `light-idle` before `light`, but `dark` before `dark-idle`. Pick one plz?